workflows/transport/common_transport.py

```diff
@@ -443,3 +444,16 @@ def _mangle_for_receiving(message):
         """Function that any message will pass through before it being forwarded to
         the receiving subscribed callback functions."""
         return message
+
+
+def json_serializer(obj):
+    """A helper function for JSON serialisation, where it can be used as
+    the default= argument. This function helps the serialiser to translate
```

Comment:
We might as well be consistent in use of "z" vs "s":
```suggestion
    """A helper function for JSON serialization, where it can be used as
    the default= argument. This function helps the serializer to translate
```